docs/source/roadmap.md

```diff
@@ -0,0 +1,287 @@
+# _starfish_ Roadmap
+This document describes the features and timeline for current work on the _starfish_ package.
+
+The purpose of this document is to describe the deliverables that we intend to deliver by June 2019. To accomplish this, we first outline our primary [use cases](#use-cases). With these to guide us, we then define milestones for releases that support these needs [**what we need to build**](#what-we-need-to-build:-starfish-0.1.0), and candidate features which we may consider building in the future [**what we can build later**](candidates-features-for-0.2.0+).
```

Comment:
Potential point of confusion: What is the significance of June 2019?  If this is obvious to everyone who might read this document, we are good.  Otherwise, it's worth expanding.